### PR TITLE
[DX][WebProfiler] #36279 Use distinct color for deprecations

### LINKED ISSUE #36279: [DX][WebProfiler] Use distinct color for deprecations
**Description**  
At the moment there are three log message types shown in the toolbar: error, warning and deprecation. Error is color coded as red, warning and deprecation both as yellow.

I think it would be helpful to introduce an additional color when only deprecations are triggered. This would lead to a better DX as it would upgrade warnings, as  they could be easier distinguished from deprecations, which may exist (for one reason or another) for quite a while.

I did some proof of concept and could provide it as a PR for starters, if there is any interest in this improvement.

### PR DESCRIPTION
**Pull request by samimussbach**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no 
| Tickets       | Fix #36279
| License       | MIT

This PR provides a new distinct color in ProfilerToolbar when only deprecations are triggered. 
This would improve DX as it would upgrade warnings, as they could be easier distinguished from deprecations, which may exist (for one reason or another) for quite a while and may lead to devs ignoring yellow colors (aka "real" warnings) in this profiler element

[] Should any tests be added? How?
[] Under which Version the Changelog should be updated?

### CODE REVIEW

**Comment by derrabus** [comment]
Interesting idea. Could you provide a screenshot of how the profiler looks like with your changes applied?

**Comment by samimussbach** [comment]
Profiler with deprecations only: 
image
image
image

Profiler with deprecations and warnings:
image
image
image

**Comment by wouterj** [comment]
As said in the issue, I think this is a great idea.

However, about the color: What about using a more dimmed color instead? The idea is that deprecations will be in the toolbar for a long time, so I think it should be informative without attracking attention.

For instance, this (using `#065179` - switched to blue as Nicolas suggested a blue-ish color):

image

**Comment by samimussbach** [comment]
I'm totally fine with almost every color that could be distinguished. I am by no means a designer

**Comment by wouterj** [comment]
Yeah, me neither. Let's see if @javiereguiluz maybe has some ideas about the color? (As he also created the original color palette)

**Comment by javiereguiluz** [comment]
Instead of introducing a new "visual state" in the toolbar, I'd do this:

* If this is not that important, don't change the background. The icon with the number is enough to know that something needs to be taken care of;
* If this is truly important, use the yellow background we use in the rest of panels to tell the usr that they must look at that soon.

**Comment by samimussbach** [comment]
I updated the PR as you suggested. It has the nice side effect of a much smaller code change

Profiler with deprecation only:
image
image
image

**Review by wouterj — APPROVED** [review]
I like it a lot!

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig [hunk 1]
```diff
@@ -32,7 +32,7 @@
 {% endblock %}
 
 {% block menu %}
-    <span class="label label-status-{{ collector.counterrors ? 'error' : collector.countdeprecations or collector.countwarnings ? 'warning' }} {{ collector.logs is empty ? 'disabled' }}">
+    <span class="label label-status-{{ collector.counterrors ? 'error' :  collector.countwarnings ? 'warning' : 'none' }} {{ collector.logs is empty ? 'disabled' }}">
```
Comment (ro0NL) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig:35]: duplicate space :angel:

**Review by javiereguiluz — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
@samimussbach Can you rebase on current master (to get rid of the merge commits)?

**Review by stof — APPROVED** [review]

**Comment by samimussbach** [comment]
@fabpot I tried to rebase but I had some really weird problems I could not resolve. I did a new PR #37105 that duplicates this one and could be merged.
Sorry to mess with the process.

**Pull request closed without merging**